.github/workflows/continuous-integration.yaml

```diff
@@ -40,8 +40,8 @@ jobs:
     name: Unit tests
     runs-on: ubuntu-latest
 
-    # max run time 10 minutes
-    timeout-minutes: 10
+    # max run time 15 minutes
```

Comment:
Increasing due to observed new timeouts. Didn't think anything here changed wrt unit tests, so maybe there's just incremental increase over time.

Reply:
Looks like this worked: unit test suite passed in bit over 13 minutes. But there's definitely some increase for time spend.

Reply:
Run time is back to 8-10 minutes, but I think it is good to allow 15 minutes just so that slightly increased latency will not cause transient failures.